### ISSUE TITLE
Swift 3 - Cannot set the debug `label` property.

### ISSUE THREAD

**Issue by cjwirth** [SnapKit/SnapKit#313]
### New Issue Checklist

* [x] I have looked at the [Documentation](http://snapkit.io/docs)
* [x] I have read the [F.A.Q.](http://snapkit.io/faq)

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform                | iOS
 Platform Version        | 8.0
 SnapKit Version         | 3.0.0
 Integration Method      | CocoaPods
 

### Issue Description

Since the `view.snp` property is a computed property, it's immutable. That's not inherently a bad thing, but it now you can't set the debug label:

```Swift
view.snp.label = "MyAwesomeView" // Cannot assign to property: `snp` is a get-only property
```

While it's not something that is absolutely necessary, it's something that stopped working when upgrading to 3.0.0.

### Workaround

It seems that, because of the way it's implemented, the returned struct still has a reference to the original view that you want to store the `label` onto. So it seems like something like this will work:

```Swift
// Old Style
view.snp_label = "MyAwesomeView"

// New Style Workaround
var viewSnp = view.snp
viewSnp = "MyAwesomeView"
```

I haven't tried it yet, but it seems like it will work, but it still takes two lines instead of one...

**Comment by robertjpayne** [comment]
@cjwirth thanks for this! I'll look into it probably just an oversight!

**Comment by robertjpayne** [comment]
@cjwirth should be fixed in 3.0.1 that just went out. But I had to change it to a function call unfortunately so you'll need to use `setLabel("")`

**Comment by cjwirth** [comment]
Awesome! 😎 

Thanks for the quick response. It's not so bad calling a function instead of setting a property :)
